docs/tutorials/zed-primer.md

```diff
@@ -1,67 +1,322 @@
+pool key, sorted in descending order:
+
+```bash
+$ zed create -orderby created_at:desc prs
+```
+=>
+```
+pool created: prs <unique pool ID>
+```
+
+Using `zed ls` we can view all the pools in the lake:
+
+
+```bash
+$ zed ls
+```
+=>
+```
+prs <pool_id> key created_at order desc
+```
+
+Let's add some pull request data I've prefetched from the GitHub API
+[here](github1.zng):
+
+```bash
+$ zed load -use prs github1.zng
+```
+=>
+```
+<commit_id> committed
+```
+
+Our data has been committed. The `-use prs` argument in `zed load` tells
+`zed` to load our data into the `prs` pool.
+
+## Querying our data
+
+With our data now loaded let's run a quick `count()` query to verify that we have
+the expected data. To do this we'll use the `zed query` command. To those
+familiar with [`zq`](../zq/README.md), `zed query` operates similarly except
+it doesn't accept file input arguments since you'll be querying against
+pools.
+
+```bash
+$ zed query -use prs 'count()'
+```
+=>
+```
+{count:100(uint64)}
+```
+
+It's looking good so far, but let's do something more interesting. First let's use
+the `zed use` command to set `prs` as our default pool so we don't have to type
+the `-use` argument every time we operate on this pool.
+
+```bash
+$ zed use prs
+```
+
+We can run an aggregation to see who has created the most PRs during the time range
+of this first data set:
+
+```bash
+$ zed query 'count() by user:=user.login | sort -r count'
+```
+=>
+```
+{user:"mccanne",count:40(uint64)}
+{user:"mattnibs",count:23(uint64)}
+{user:"aswan",count:20(uint64)}
+{user:"henridf",count:9(uint64)}
+{user:"nwt",count:5(uint64)}
+{user:"philrz",count:3(uint64)}
+```
+
+A productive few weeks for McCanne!
+
+We can use the `min` / `max` aggregations to see the time range of our data set:
+
+```bash
+$ zed query -Z 'min(created_at), max(created_at)'
+```
+=>
+```
+{
+    min: 2019-11-11T19:50:46Z,
+    max: 2019-12-05T16:56:57Z
+}
+```
+
+That's not a lot of data, so let's add some more.
+
+## Adding additional data
+
+Additional data can be added to our pool by running `zed load` on our second
+[data set](github2.zng):
+
+```bash
+$ zed load github2.zng
+```
+
+Running our `min(created_at), max(created_at)` query we'll see that we now have
+almost two years of pull requests:
+
+```bash
+$ zed query -Z 'min(created_at), max(created_at)'
+```
+=>
+```
+{
+    min: 2019-11-11T19:50:46Z,
+    max: 2021-09-19T19:31:43Z
+}
+```
+
+Now let's run a bucketed aggregation to count approximate PRs per month (specifically, PRs
+bucketed in 12 equal spans of a year):
+
+```
+$ zed query 'count() by ts:=bucket(created_at, 1y/12) | sort ts'
+```
+=>
+```
+{ts:2019-10-20T04:00:00Z,count:28(uint64)}
+{ts:2019-11-19T14:00:00Z,count:123(uint64)}
+{ts:2019-12-20T00:00:00Z,count:72(uint64)}
+{ts:2020-01-19T10:00:00Z,count:102(uint64)}
+{ts:2020-02-18T20:00:00Z,count:114(uint64)}
+{ts:2020-03-20T06:00:00Z,count:111(uint64)}
+{ts:2020-04-19T16:00:00Z,count:137(uint64)}
+...
```

Comment:
Maybe include the next row so the appearance of 2020-05-19T16:00:00Z below is less magical?

Reply:
Yes!